doc/architecture/trace_file_naming.adoc

```diff
@@ -76,3 +82,9 @@ The recommended file name is:
 ----
 20210818T150542Z_sv_312_300_1523_highway.osi
 ----
+
+For a corresponding multi channel trace file containing `SensorView` and `GroundTruth` messages, the recommended file name is:
+
+----
+20210818T150542Z_multi_312_300_1523_highway.mcap
+----
```

Comment:
I know, you would like to move the naming convention discussion to a separate issue. But I am still very much against using a timestamp in a simulation data file name. I also know, it is "just" a recommendation. But I would still not even recommend it.

Reply:
See discussion above on `zero_time`: I don't think that the missing relationship to some real-time is as common as you assume; that said I can certainly see making the timestamp optional for certain use cases could be a thing; but as you say, I think this needs to be handled in a seprate PR anyways.

Reply:
Thats odd. I regularly see trace files from OEMs, Tier1s, tool vendors and research institutes without the timestamp.

Reply:
I regularly see trace files with wrong moving object coordinate usage, missing mandatory fields, etc., too. Does not mean this is good or correct usage. And if it works by accident, it still works.

Reply:
Not using something or using something in the wrong way are completely different things.

Reply:
Moves to #843.